docs/integrations/persisting-store-data.md

```diff
@@ -395,29 +424,49 @@ unsub()
 
 ## Hydration and asynchronous storages
 
-To explain what's the "cost" of asynchronous storages, you need to understand what's hydration.
-In a nutshell, hydration is the process of retrieving the persisted state from the storage and merging it with the current state.
+To explain what is the "cost" of asynchronous storages,
```

Comment:
I introduced [Semantic line breaks](https://sembr.org/)
for better maintainability of those paragraphs.

Reply:
That is a really interesting read! Thank you for sharing (and introducing the change)